pkg/cmd/pr/review/review.go

```diff
@@ -273,8 +273,7 @@ func reviewSurvey(io *iostreams.IOStreams, editorCommand string) (*api.PullReque
 	}
 
 	if len(bodyAnswers.Body) > 0 {
-		style := markdown.GetStyle(io.DetectTerminalTheme())
```

Comment:
Looks like `io.DetectTerminalTheme()` got dropped here, is it no longer necessary?